new_samples/README.md

```diff
@@ -0,0 +1,32 @@
+# Cadence Samples
+These are some samples to demonstrate various capabilities of Cadence client and server.  You can learn more about cadence at:
+* Cadence: https://github.com/uber/cadence
+* Cadence Client: https://github.com/uber-go/cadence-client
+
+## Prerequisite
+0. Install Cadence CLI. See instruction [here](https://cadenceworkflow.io/docs/cli/).
+1. Run the Cadence server: https://github.com/uber/cadence/blob/master/README.md
+2. Register the `cadence-samples` domain: https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
+
+
+
+## Steps to run samples
+Simply start the go binary by running
+```bash
+go run main.go
+```
+The worker will listen to `localhost:3000`. Then you may interact with the samples using Cadence CLI.
+Here are a list of example CLIs you may use to play with the samples.
+
+### HelloWorld workflow
+This workflow takes an input message and greet you as response. Try the following CLI
+```bash
+cadence --env development \
+--domain cadence-samples-domain \
+workflow start \
+--workflow_type cadence_samples.HelloWorldWorkflow \
+--tl cadence-samples-worker \
+--et 60 \
+--input '{"message":"Uber"}'
```

Comment:
```suggestion
cadence --env development \
  --domain cadence-samples-domain \
  workflow start \
  --workflow_type cadence_samples.HelloWorldWorkflow \
  --tl cadence-samples-worker \
  --et 60 \
  --input '{"message":"Uber"}'
```
personally I prefer to indent at least a bit, so it's clear if it's one command or multiple (without needing to read the trailing `\`, which moves all over).

but that's just personal preference, pick whatever :)

Reply:
sure